### DISCUSSION TITLE
对于图片中存在横、竖列摆放文字及换行文字有什么好的解决办法吗

### DISCUSSION THREAD

**Discussion by dong6255** [PaddlePaddle/PaddleOCR#13980] [Q&A] [answered]
![223](https://github.com/user-attachments/assets/535cded4-fd58-471c-9038-64032cb58002)

使用的是ocrv4-server，识别单行横向文字和纵向文字没什么问题，但是识别换行文字会被拆成多行。想实现这个功能是需要让检测模型拆开进行吗？或者有什么其他好的解决办法吗？

**Comment by GreatV** [comment, marked as answer]
在得到检测和识别结果后，可以根据文本框的位置信息（如上下文位置、行间距）编写后处理代码，将属于同一段落或句子的文本行合并。
